llvm/utils/TableGen/Common/OptEmitter.cpp

```diff
@@ -41,6 +41,8 @@ static int StrCmpOptionName(const char *A, const char *B) {
 
 // Returns true if A is ordered before B.
 bool CompareOptionRecords(const Record *A, const Record *B) {
+  if (A == B)
```

Comment:
Changing from array_pod_sort() to std::sort() seems to expose this platform specific difference, but this change should be safe in general.